docs/make.jl

```diff
@@ -1,3 +1,5 @@
+import Pkg;
+Pkg.add("Documenter")
```

Comment:
You don't need these lines if you already have it in `docs/Project.toml`, right?

Reply:
I don't know. Couldn't get the package manager to handle it.